tests/components/tplink/__init__.py

```diff
@@ -250,8 +329,8 @@ def _mocked_energy_features(
     if power is not None:
         feats.append(
             _mocked_feature(
-                power,
                 "current_consumption",
+                value=power,
                 name="Current consumption",
                 type_=Feature.Type.Sensor,
                 category=Feature.Category.Primary,
```

Comment:
Should we also define the rest of the expected types inside the `features.json`?

Reply:
Which ones? I've defined for all the primitive platforms we're supporting.

Reply:
Now updated to not override any of the fixture values except `value`